doc/release-notes/release-5.sgml

```diff
@@ -151,6 +152,23 @@ Most user-facing changes are reflected in squid.conf (see below).
    yet do TLS-in-TLS.
 
 
+<sect1>OpenSSL 3.0 Support
+<p>Squid-5.7 and later now support build with OpenSSL 3.0.
+
+<p>This version of Squid does not add any of the new features provided by
+   OpenSSL 3.0. It only contains support for features already supported by prior
+   versions of Squid using new APIs provided by OpenSSL 3.0.
+
+<p>Notably the libssl custom Engine feature has been deprecated by OpenSSL 3.0
+   and new Providers replacement is not supported by this Squid.
+
+<p>OpenSSL 3.0 license changes also mean a change in compatibility with GPLv2+ Squid
+   is distributed under. The previous restrictions on distribution are no longer
+   applicable, if and only if, Squid is built to use both libssl and libcrypto
+   from OpenSSL 3.0 or later. Care should still be taken to ensure distributed
+   binariues meet the appropriate licenses per their build.
```

Comment:
This paragraph contains too many controversial opinions on legal matters and corresponding advice. I disagree with some of those opinions/advice, and I feel the overall tone is too close to [FUD](https://en.wikipedia.org/wiki/Fear,_uncertainty,_and_doubt), but I would very much prefer not to fight for consensus in those details because those fights tend to waste a lot of time and because a ChangeLog should not tell what (we think) the licenses mean. I suggest simplifying this to focus on the facts:

```suggestion
<p>OpenSSL 3.0 uses new licensing terms.
```

... or something like that.

The above suggestion will also fix spelling in the above paragraph.